01_01_Basic_Rules.txt

```diff
@@ -736,44 +736,57 @@ push you below the minimum miss chance.
-opponent, they are both considered to be in "melee combat," and occupy spaces 
-within reach of one of the combatant's melee weapons. At this point (unless one 
-character has a CQB weapon) both characters are assumed to be using their fists 
-or other melee weapons. If a character wishes to leave melee combat, they incur 
-an "Opportunity Attack."  
+	A character of standard size occupies 1m of space. This is not the physical 
+space occupied, but rather both the area effectively controlled by the character and 
+the minimum necessary to effectively fight.
+
+	Two characters engaged in melee combat are considered to be in adjacent 1m 
+spaces, not in the same 1m space. Fights don't generally start face-to-face unless 
+you're at a bar.
+
+	Once a character has attempted to attack another at melee range (1m), 
+they are both considered to be in melee combat. They can make attacks against 
+each other with unarmed strikes, melee weapons (improvised or not), or CQB guns 
+as normal. Attacking a target more than 1m away, using a non-CQB weapon, or 
+moving away from the melee (under the character's own power: explosion knockback 
+does not count) will incur an Opportunity Attack as described below.
 	
 Opportunity Attack:
-	A free attack* that is triggered when an opponent leaves melee combat or 
-walks adjacent (within melee range) to a character that is aware of their 
-opponent. Characters also incur Opportunity Attacks by performing a non-combat 
-skill check when locked in melee combat (such as trying to repair their own 
-armor) or by trying to shoot a different opponent with a non-CQB weapon.
+	A reactionary attack that is triggered when an opponent leaves melee combat, 
+walks past (within melee range) an alert character, performing a skill check 
+when engaged in melee combat that requires the use of the character's primary 
+manipulator limbs (so a medicine check to stabilize an ally would incur an 
+attack, but an acrobatics check to kick away a grenade would not) or by 
+trying to shoot an opponent more than 1m away while engaged in melee combat.
 
-*The free attack can be made with a CQB weapon or a melee weapon (depending on 
-if the weapon is already equipped in the character's hands). If the free attack 
-is made with a CQB weapon, and the weapon is not already aimed at the target, 
-standard Hip Firing rules apply.
+	Opportunity attacks can be made with a drawn CQB weapon or a melee weapon. 
+If the free attack is made with a CQB weapon and the weapon is not already 
+aimed at the target, standard Hip Firing rules apply.
 
 Making Melee Attacks:
-	Melee combat is relatively simple. To hit an enemy player: add your 
-dexterity combat modifier to a d10 roll plus your combat modifier from your 
-weapon/attack*. The result must be greater than the opponent's Guard DC: 
+	Melee attacks make accuracy rolls with a d10, increased by your 
+dexterity combat modifier and your weapon's accuracy modifier. To successfully 

```

Comment:
I think I forgot to mention in my last review how much I like making one D10 the standard for attack/defense rolls. Great subtle change!